### DISCUSSION TITLE
Central or Separate sccache

### DISCUSSION THREAD

**Discussion by xZanon** [mozilla/sccache#1634] [Q&A] [answered]
Hello, 

I am planning to use sccache in heterogenous environment Windows/linux, compiling using cmake and multiple versions of compiler: gcc 7, gcc9, gcc10, VS2019 ... 
My question is : Should I create separate S3_prefix for all of them, or I can use only one for all of them ?

Thank you in advance.

**Comment by Xuanwo** [comment, marked as answer]
Only one is enough. Sccache will generate unique hash for all of them.

**Reply by xZanon** [reply]
Great thank you :) 👍
